com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/AndroidAAB.java

```diff
@@ -172,37 +172,44 @@ private static void copyEngineBinary(Project project, Platform architecture, Fil
 	/**
 	* Copy Android resources per package
 	*/
-	private static File copyResources(Project project, File bundleDir, ICanceled canceled) throws IOException {
-		// create a list of resource directories, per package
+	private static File copyResources(Project project, File bundleDir, BundleHelper helper, ICanceled canceled) throws IOException {
 		Platform platform = Platform.Armv7Android;
 		File packagesDir = new File(project.getRootDirectory(), "build/"+platform.getExtenderPair()+"/packages");
+
+		File androidResDir = createDir(bundleDir, "res");
+
+		// create a list of resource directories, per package
 		List<File> directories = new ArrayList<>();
-		File packagesList = new File(packagesDir, "packages.txt");
-		if (packagesList.exists()) {
-			List<String> allLines = Files.readAllLines(new File(packagesDir, "packages.txt").toPath());
-			for (String line : allLines) {
-				File resDir = new File(packagesDir, line);
-				if (resDir.isDirectory()) {
-					directories.add(resDir);
+		if(packagesDir.exists()) {
+			File packagesList = new File(packagesDir, "packages.txt");
+			if (packagesList.exists()) {
+				List<String> allLines = Files.readAllLines(new File(packagesDir, "packages.txt").toPath());
+				for (String line : allLines) {
+					File resDir = new File(packagesDir, line);
+					if (resDir.isDirectory()) {
+						directories.add(resDir);
+					}
 				}
 			}
-		}
-		else {
-			for (File dir : packagesDir.listFiles(File::isDirectory)) {
-				File resDir = new File(dir, "res");
-				if (resDir.isDirectory()) {
-					directories.add(resDir);
+			else {
+				for (File dir : packagesDir.listFiles(File::isDirectory)) {
+					File resDir = new File(dir, "res");
+					if (resDir.isDirectory()) {
+						directories.add(resDir);
+					}
 				}
 			}
 		}
+		else {
```

Comment:
Copy resources (icons) in non-ne builds!

Reply:
Perhaps would be good to add that comment in the code?